prompts/github-copilot-custom-instructions.prompt.md

```diff
@@ -0,0 +1,18 @@
+---
+mode: "ask, edit, agent"
+tools: ["codebase", "editFiles", "problems"]
+description: "Create a GitHub Copilot Custom instructions file"
```

Comment:
This matches the title, it'd be preferable if they aren't the same.

Reply:
Any proposed text? I am blanking on how to differentiate 😄